{
  Throwable exception=null;
  try {
    super.deleteDirectory(path);
  }
 catch (  Throwable t) {
    exception=t;
  }
 finally {
    sop("deleteDirectory[SECURE]: " + path(path),exception);
  }
}
